packages/apptive_grid_form/lib/src/widgets/form_widget/form_content_widgets.dart

```diff
@@ -587,12 +589,17 @@ class FormPage extends StatelessWidget {
               );
 
               if (block != null) {
+                TextStyle? style = textBlockStyle;
+                if (style == null && block.style == FormTextBlockStyle.header) {
+                  style ??= titleStyle;
+                } else {
+                  style ??= descriptionStyle;
+                }
                 return Padding(
                   padding: textBlockPadding ?? padding,
                   child: Text(
                     block.text,
-                    style:
-                        textBlockStyle ?? Theme.of(context).textTheme.bodyLarge,
```

Comment:
As you are now using the `descriptionStyle` I think we should remove this as a parameter. (Think only having two styles to put in (Title and Description) is better as having 3 different styles can introduce way to much clutter)

Reply:
done